polaris/dataset/zarr/_utils.py

```diff
@@ -11,3 +11,22 @@ def load_zarr_group_to_memory(group: zarr.Group) -> dict:
         elif isinstance(item, zarr.Group):
             data[key] = load_zarr_group_to_memory(item)
     return data
+
+
+def check_zarr_codecs(group: zarr.Group):
+    """Check if all codecs in the Zarr group are registered."""
+    try:
+        for key, item in group.items():
+            if isinstance(item, zarr.Group):
+                check_zarr_codecs(item)
+    except ValueError as error:
+        # Zarr raises a generic ValueError if a codec is not registered.
+        # See also: https://github.com/zarr-developers/zarr-python/issues/2508
+        prefix = "codec not available: "
+        message = str(error)
+        codec_id = message.removeprefix(prefix)
+
+        if message.startswith(prefix):
```

Comment:
Minor: (for readability)

```suggestion
        if error_message.startswith(prefix):
```